aiida_quantumespresso/parsers/ph.py

```diff
@@ -56,6 +56,10 @@ def parse(self, **kwargs):
         self.emit_logs(logs)
         self.out('output_parameters', orm.Dict(dict=parsed_data))
 
+        # scheduler plugin detected a problem
+        if self.node.exit_status is not None:
```

Comment:
This should be a more specific check. You should compare the status to the actual status of the OOW exit code

Reply:
My idea here is that when an exit status is already set it means the scheduler catch a problem, what it does is propagate it up and let it be handled by the workchain handler. 
Maybe I misunderstand the documentation here https://aiida.readthedocs.io/projects/aiida-core/en/latest/topics/calculations/usage.html?highlight=scheduler%20handler#scheduler-errors ?

Reply:
That is completely true, but there could be a variety of exit statuses being set and we might not all want to treat them the same way. So what I am suggesting is to do this instead:
```python
# If the scheduler detected OOW, simply keep that exit code by not returning anything more specific.
if self.node.exit_status == PhCalculation.exit_codes.ERROR_SCHEDULER_OUT_OF_WALLTIME:
    return
```

The documentation does indeed not set the right example here. I will open a PR to fix it.